build-scripts/hack/update-component-versions.py

```diff
@@ -134,6 +140,25 @@ def update_component_versions(dry_run: bool):
             pull_helm_chart()
 
 
+def update_go_version(dry_run: bool):
```

Comment:
Do we need to include any sort of testing for this? Or maybe I'm just being paranoid :D

Reply:
so, what do you have in mind @HomayoonAlimohammadi ?  As far as i see it -- none of this update-component-versions code is tested (nor is it in the product -- its just build script stuff)

I guess we can start testing it (linting/formatting...etc).  Maybe need a card and another task for it?  I don't see it being critical b/c it doesn't impact the final product

Reply:
+1 let's get this merged.